### PR TITLE
Fix some minor issues

### PR DESCRIPTION
**Pull request by ischoegl**
**Changes proposed in this pull request**

- Ensure that `scons test` triggers check for `pytest` (see #1434)
- Fix author order in `AUTHORS`
- Update copyright information
- Bump version to `3.0.0a4`

**Checklist**

- [x] The pull request includes a clear description of this code change
- [x] Commit messages have short titles and reference relevant issues
- [x] Build passes (`scons build` & `scons test`) and unit tests address code coverage
- [x] Style & formatting of contributed code follows [contributing guidelines](https://github.com/Cantera/cantera/blob/main/CONTRIBUTING.md)
- [x] The pull request is ready for review

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -56,6 +56,7 @@ Travis Sikes (@tsikes)
 Harsh Sinha (@sin-ha)
 David Sondak
 Raymond Speth (@speth), Massachusetts Institute of Technology
+Su Sun (@ssun30), Northeastern University
```
Comment (bryanwweber) [inline AUTHORS:59]: What order are you putting this in?

Comment (ischoegl) [reply AUTHORS:59]: Per top of file:
> Below is a partial, alphabetical list of developers and contributors to Cantera over the years.

Comment (bryanwweber) [reply AUTHORS:59]: But my name is immediately below this? I'm all for alphabetizing but it should be done throughout IMO

**Comment by speth** [comment]
Can I also suggest a bump to `3.0.0a4`? It's been 5 months and 200 commits since the last one. I meant to do this in my last PR but forgot.

File: AUTHORS (hunk 1 above)
Comment (ischoegl) [reply AUTHORS:59]: Hmm - your name is between Walker and Wehrfritz in the document I am looking at? My reading is that the list is alphabetical by last name?

**Comment by ischoegl** [comment]
> Can I also suggest a bump to `3.0.0a4`? It's been 5 months and 200 commits since the last one. I meant to do this in my last PR but forgot.

It's true that we should bump soon; I was hoping to get #1426 in before, but I know that the review of this one requires some bandwidth.

File: AUTHORS (hunk 1 above)
Comment (bryanwweber) [reply AUTHORS:59]: Oh, darn it 😔 you're right, thank you!

**Comment by ischoegl** [comment]
@speth ... just added the suggested version bump.

**Review by speth — APPROVED** [review]
Thanks, @ischoegl. This looks good to me.

**Pull request merged**